src/driver/alexnet.cpp

```diff
@@ -61,7 +61,7 @@ migraphx::program alexnet(unsigned batch) // NOLINT(readability-function-size)
     migraphx::op::relu relu19;
     auto mx19 = mm->add_instruction(relu19, mx18);
     migraphx::op::pooling pooling20;
-    pooling20.mode    = "max";
+    pooling20.mode    = migraphx::op::pooling_mode::kMax;
```

Comment:
Looks like clang tidy doesn't accept the variable names `kMax` and `kAvg`. Since we are scoping it to pooling mode only, would it be fine to just have `Max` and `Avg`?